tests/e2e/cucumber/features/app-provider/officeSuites.feature

```diff
@@ -231,3 +231,38 @@ Feature: Integrate with online office suites like Collabora and OnlyOffice
       | usingFolderLink.odt |
     Then "Alice" should see the content "OpenDocument Content" in editor "Collabora"
     And "Alice" logs out
+
+
+  Scenario: create files from office templates
+    Given "Alice" uploads the following local file into personal space using API
+      | localFile     | to            |
+      | Template.dotx | Template.dotx |
+      | Template.ott  | Template.ott  |
+
+    When "Alice" creates a file from template file "Template.dotx" via "OnlyOffice" using the sidebar panel
+    Then "Alice" should see the content "As a user I want to create a document by clicking on a template file" in editor "OnlyOffice"
+    And "Alice" closes the file viewer
+
+    When "Alice" creates a file from template file "Template.ott" via "Collabora" using the context menu
```

Comment:
when running services from `web` we see only **onlyoffice** but added tests for **collabora** too. Any env missing in docker-compose? If so, please add.
![Screenshot from 2024-11-28 11-37-47](https://github.com/user-attachments/assets/fb72f4dd-09c6-4c30-bb29-3412d3fc0049)

Reply:
I also found the same while working on oCIS but your tests seem to create a file from a template using Collabora. How is this possible, or did I understand wrong?